### ISSUE TITLE
Revanced yt asking for microg with magisk module 

### ISSUE THREAD

**Issue by oktest145** [j-hc/revanced-magisk-module#279]

**Comment by oktest145** [comment]
![Screenshot_20230714-212659_ReVanced Extended](https://github.com/j-hc/revanced-magisk-module/assets/83488344/4a691666-d558-46e8-a560-ee99fbbb0217)
![Screenshot_20230714-212652_Magisk](https://github.com/j-hc/revanced-magisk-module/assets/83488344/26a4beac-9a1c-4a90-90b9-0247119d1c2b)

**Comment by sixstrings** [comment]
You're using the module from NoName-exe which is not related to the one of J-hc.
So you're report should go to his repo.

**Comment by j-hc** [comment]
you are using a module from one of my forks not from this repo.
refrain from making false bug reports or you will be blocked

**Comment by j-hc** [comment]
> You're using the module from NoName-exe which is not related to the one of J-hc. So you're report should go to his repo.

No, that module belongs to me too. People for some reason downloads from those outdated forks of mine and reports the bugs here.

**Comment by sixstrings** [comment]
I'm sorry. Just wanted to help.
